FhirToDataLake/test/Microsoft.Health.AnalyticsConnector.Core.UnitTests/Jobs/FhirToDataLakeOrchestratorJobTests.cs

```diff
@@ -43,6 +47,25 @@ public class FhirToDataLakeOrchestratorJobTests
         private static readonly DateTimeOffset TestEndTime = new DateTimeOffset(2020, 11, 1, 0, 0, 0, TimeSpan.FromHours(0));
 
         private static readonly List<TypeFilter> TestResourceTypeFilters = new List<TypeFilter> { new TypeFilter("Patient", null) };
+        private static readonly List<TypeFilter> TestFourResourceTypeFilters = new List<TypeFilter> { new TypeFilter("Patient1", null), new TypeFilter("Patient2", null), new TypeFilter("Patient3", null), new TypeFilter("Patient4", null) };
+        private static readonly int ResourceCountPerDayPerJob = (int)(JobConfigurationConstants.HighBoundOfProcessingJobResourceCount / (TestEndTime - TestStartTime).TotalDays);
+
+        public static IEnumerable<object[]> GetResourceCountForEachResourceType()
+        {
+            yield return new object[] { new Dictionary<string, int>() { { "Patient", 400000 } }, 1 };
```

Comment:
Can we add more cases to cover zero jobs or many small jobs / different resource types?

Reply:
Added here, thanks.